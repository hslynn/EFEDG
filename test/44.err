Running /share/soft/intel_2018_update1/compilers_and_libraries_2018.1.163/linux/mpi/intel64/bin/mpiexec.hydra -np 44 -ppn 22 -hostfile /share/home/hslynn/.lsbatch/1546157612.208233.hostfile -cleanup ./simplest
