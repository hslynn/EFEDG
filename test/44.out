Sender: LSF System <lsfadmin@a6u12n04>
Subject: Job 208233: <44> in cluster <hpc.lsec.cc.ac.cn> Done

Job <44> was submitted from host <ln02> by user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
Job was executed on host(s) <36*a6u12n04>, in queue <batch>, as user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
                            <8*c1u17n01>
</share/home/hslynn> was used as the home directory.
</share/home/hslynn/Einstein_DG/test> was used as the working directory.
Started at Sun Dec 30 16:20:04 2018
Results reported on Sun Dec 30 16:20:08 2018

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
mpijob -t mvapich2 ./simplest
------------------------------------------------------------

Successfully completed.

Resource usage summary:

    CPU time :                                   58.85 sec.
    Max Memory :                                 726 MB
    Average Memory :                             200.00 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Processes :                              28
    Max Threads :                                29

The output (if any) follows:

Parallel Hierarchical Grid (version 0.9.4).
Repartition mesh, load imbalance: 1.03125
729 DOF, 384 elements, 44 submeshes, load imbalance: 1.03125
  DOF: 729, unknowns: 729, Dirichlet bdry: 386
  nits = 20, L2 error = 2.295e-02, indicator = 1.204e+01, mem = 38.39MB
  Wall time: 5.875e-02
1241 DOF, 768 elements, 44 submeshes, load imbalance: 1.03125
  DOF: 1241, unknowns: 1241, Dirichlet bdry: 386
  nits = 29, L2 error = 1.626e-02, indicator = 9.157e+00, mem = 43.77MB
  Wall time: 7.766e-03
2009 DOF, 1152 elements, 44 submeshes, load imbalance: 1.03125
  DOF: 2009, unknowns: 2009, Dirichlet bdry: 770
  nits = 34, L2 error = 3.494e-03, indicator = 4.963e+00, mem = 43.91MB
  Wall time: 7.928e-03
Repartition mesh, load imbalance: 1.02778
2645 DOF, 1584 elements, 44 submeshes, load imbalance: 1.02778
  DOF: 2645, unknowns: 2645, Dirichlet bdry: 770
  nits = 38, L2 error = 3.436e-03, indicator = 4.483e+00, mem = 44.33MB
  Wall time: 1.523e-02
Repartition mesh, load imbalance: 1.01215
3953 DOF, 2304 elements, 44 submeshes, load imbalance: 1.01215
  DOF: 3953, unknowns: 3953, Dirichlet bdry: 1538
  nits = 41, L2 error = 3.117e-03, indicator = 3.546e+00, mem = 48.45MB
  Wall time: 1.832e-02
6129 DOF, 3984 elements, 44 submeshes, load imbalance: 1.18173
  DOF: 6129, unknowns: 6129, Dirichlet bdry: 1538
  nits = 46, L2 error = 2.621e-03, indicator = 2.852e+00, mem = 48.85MB
  Wall time: 9.386e-03
Repartition mesh, load imbalance: 1.0026
9009 DOF, 6144 elements, 44 submeshes, load imbalance: 1.0026
  DOF: 9009, unknowns: 9009, Dirichlet bdry: 1538
  nits = 51, L2 error = 1.732e-03, indicator = 2.213e+00, mem = 49.27MB
  Wall time: 1.278e-02
14769 DOF, 9600 elements, 44 submeshes, load imbalance: 1.05417
  DOF: 14769, unknowns: 14769, Dirichlet bdry: 2690
  nits = 62, L2 error = 1.146e-03, indicator = 1.555e+00, mem = 49.72MB
  Wall time: 1.484e-02
22041 DOF, 14400 elements, 44 submeshes, load imbalance: 1.06333
  DOF: 22041, unknowns: 22041, Dirichlet bdry: 3842
  nits = 70, L2 error = 4.832e-04, indicator = 1.065e+00, mem = 50.58MB
  Wall time: 2.103e-02
31209 DOF, 21120 elements, 44 submeshes, load imbalance: 1.02917
  DOF: 31209, unknowns: 31209, Dirichlet bdry: 5378
  nits = 71, L2 error = 3.251e-04, indicator = 8.291e-01, mem = 51.03MB
  Wall time: 2.631e-02
51329 DOF, 36096 elements, 44 submeshes, load imbalance: 1.06416
  DOF: 51329, unknowns: 51329, Dirichlet bdry: 6146
  nits = 77, L2 error = 2.469e-04, indicator = 6.485e-01, mem = 51.92MB
  Wall time: 4.035e-02
82433 DOF, 57408 elements, 44 submeshes, load imbalance: 1.03853
  DOF: 82433, unknowns: 82433, Dirichlet bdry: 8450
  nits = 94, L2 error = 1.776e-04, indicator = 4.819e-01, mem = 53.04MB
  Wall time: 5.776e-02


PS:

Read file <44.err> for stderr output of this job.

